src/ctelnet.cpp

```diff
@@ -1914,7 +1914,12 @@ void cTelnet::handle_socket_signal_readyRead()
         }
         else
         {
-            if( ch != '\r' && ch != 0 ) cleandata += ch;
+            if( ch == TN_BELL )
+            {
+                // flash taskbar for 3 seconds
+                QApplication::alert( mudlet::self(), 3000 );   
+            }
+            else if( ch != '\r' && ch != 0 ) cleandata += ch;
```

Comment:
It would be. @kbasten, think you're up for adding the Lua API command? We can help guide you through it. It's OK if not!